### ISSUE TITLE
Presets

### ISSUE THREAD

**Issue by kitze** [kitze/sizzy#4]
- [ ] Allow storing a current preset of the view (filters, zoom levels, orientations)
- [ ] Show a list of the stored presets

**Comment by SaraVieira** [comment]
@kitze Maybe local storage for this ?

**Comment by kitze** [comment]
@SaraVieira yeah that's the plan, I just need to define some things about how the stored data will look like, so it's early for this.

**Comment by SaraVieira** [comment]
@kitze If you need some help when implementing this just ping me 👍

**Comment by kitze** [comment]
@SaraVieira alright, thanks!
